src/app/App.js

```diff
@@ -8,7 +8,6 @@ import React from 'react';
 // Gutenberg imports
```

Comment:
I mistakenly pushed changes to this file directly to master 😱
This is reverting those changes.